src/runtime/HalideRuntimeHexagonHost.h

```diff
@@ -106,6 +106,10 @@ typedef struct {
 // This is deprecated.
 typedef halide_hexagon_power_t halide_hvx_power_perf_t;
 
+/** Set fastRPC thread priority and staksize. Set thread params before making
+ * any RPC calls. halide_hexagon_set_thread_params. */
+extern int halide_hexagon_set_thread_params(void *user_context, int priority, int stack_size);
```

Comment:
Maybe this and halide_hexagon_set_thread_priority should be documented together so they can share the more detailed comment about priorities?

Reply:
I don't know if ``halide_hexagon_set_thread_priority`` mentions that lower int values mean higher priority. We should mention it in the comments here regardless.

Reply:
halide_hexagon_set_thread_priority does mention about this: "Smaller number for higher priority"